test/cbmc/proofs/Shadow_MatchTopic/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2021 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=Shadow_MatchTopic_harness
+PROOF_UID = Shadow_MatchTopic
+
+# The topic length is bounded to reduce the proof run time. This length bounds
+# the time to look for the thing name in the topic string. The minimum length
+# of the topic string is the length of the SHADOW_PREFIX ("$aws/things/")
+# plus the maximum Shadow Message Type length ("/shadow/update/documents").
```

Comment:
What about the name of the thing?  They must be [at least one character in length.](https://docs.aws.amazon.com/iot/latest/apireference/API_CreateThing.html)

Also, this vallue does not agree with the definition of `TOPIC_STRING_LENGTH_MAX` on line 33 (36 vs. 50).

Reply:
Thanks for the comment. Reading again the comments, I see the issue. The difference is allotted for ThingName Length.(50-37). I will update the comment to indicate that.